server/src/main/java/keywhiz/service/resources/admin/SecretsResource.java

```diff
@@ -248,6 +252,9 @@ public Response createSecret(@Auth User user, @Valid CreateSecretRequestV2 reque
       if (request.metadata() != null) {
         extraInfo.put("metadata", request.metadata().toString());
       }
+      if (request.owner() != null) {
+        extraInfo.put("owner", request.owner());
+      }
```

Comment:
There's a separate audit log statement for every secret creation which I only discovered after poking around in the logs during debugging. I figured it couldn't hurt to add the owner field there as well.